go/arrow/csv/writer_test.go

```diff
@@ -146,7 +146,17 @@ func TestCSVWriter(t *testing.T) {
 	defer rec.Release()
 
 	w := csv.NewWriter(f, schema, csv.WithComma(';'), csv.WithCRLF(false))
-	err := w.Write(rec)
+	err := w.Flush()
```

Comment:
we should probably put this *after* the call to `w.Write(rec)` :)

also, please add a call to `Flush` inside the example.